### ISSUE TITLE
visdrone +CSQ ,AP is 27.3，which is different from 28.3 in paper

### ISSUE THREAD

**Issue by syddpy666** [ChenhongyiYang/QueryDet-PyTorch#68]
Hello, I use the visdrone data set, the batch is set to 16, the iter is set to 50000, the step is set to (30000, 40000), the learning rate is set to 0.001, and the test AP after CSQ is turned on is 27.3, which is different from the 28.3 in the paper. Is there any setting that I am missing, or is there any deviation from the settings used in the paper?

Also, using the model_final.pth generated by detectron2, is it the best AP weight? Is it the AP gap caused by model_final.pth not being the best weight?

Looking forward to your recovery, thanks! !

**Comment by Disciple7** [comment]
I suggest lr to be 0.01, which may lead to a better result. With batchsize=8 I barely met the question that the gradient exploded, and result shows it works. The default setting is strongly suggested. See #65 and this may be useful

**Comment by syddpy666** [comment]
> I suggest lr to be 0.01, which may lead to a better result. With batchsize=8 I barely met the question that the gradient exploded, and result shows it works. The default setting is strongly suggested. See #65 and this may be useful

Thank you very much for your suggestion. I changed the training settings before because I encountered a gradient explosion. I will follow your suggestion and use the default settings to re-run the experiment.

**Comment by syddpy666** [comment]
> I suggest lr to be 0.01, which may lead to a better result. With batchsize=8 I barely met the question that the gradient exploded, and result shows it works. The default setting is strongly suggested. See #65 and this may be useful

Thanks for your suggestion, I used the default settings for training, and then tested with CSQ on the visdrone dataset, and the AP reached 32.499%. This shows that QUERYDET is really an excellent job.
![image](https://user-images.githubusercontent.com/86947992/226779553-ff727698-b787-494f-9915-061d49a97dc2.png)

**Comment by judycpChen** [comment]
> Hello, I use the visdrone data set, the batch is set to 16, the iter is set to 50000, the step is set to (30000, 40000), the learning rate is set to 0.001, and the test AP after CSQ is turned on is 27.3, which is different from the 28.3 in the paper. Is there any setting that I am missing, or is there any deviation from the settings used in the paper?
> 
> Also, using the model_final.pth generated by detectron2, is it the best AP weight? Is it the AP gap caused by model_final.pth not being the best weight?
> 
> Looking forward to your recovery, thanks! !

hello, I wonder what your configuration looks like
